constants/constants.go

```diff
@@ -139,19 +126,79 @@ var PastelParamsCheckSums = map[string]string{
 
 // ChromeDownloadURL - The download url of chrome
 var ChromeDownloadURL = map[OSType]string{
-	Windows: "https://download.pastel.network/latest/pasteld/pastel-win64-rc5.1.zip",
+	Windows: "",
 	Linux:   "https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb",
-	Mac:     "https://download.pastel.network/latest/pasteld/pastel-osx-rc5.1.tar.gz",
+	Mac:     "",
 	Unknown: "",
 }
 
 // ChromeExecFileName - The download filename of chrome executable
 var ChromeExecFileName = map[OSType]string{
-	Windows: "pastel-win64-",
+	Windows: "",
 	Linux:   "google-chrome.deb",
-	Mac:     "pastel-osx-",
+	Mac:     "",
 	Unknown: "",
 }
 
 // PortList - PortList to open in install supernode
 var PortList = []string{"9933", "19933", "4444", "14444"}
+
+// PastelRQServiceArchiveName - The name of the pastel rqservice files
+var PastelRQServiceArchiveName = map[OSType]string{
+	Windows: "rqservice-win-amd64.zip",
+	Linux:   "rqservice-ubuntu20.04-amd64.zip",
+	Mac:     "rqservice-darwin-amd64.zip",
+	Unknown: "",
+}
+
+// PastelRQServiceExecName - The name of the rqservice executable files
+var PastelRQServiceExecName = map[OSType]string{
+	Windows: "rq-service-win10-x64.exe",
+	Linux:   "rq-service-ubuntu20.04-x64",
+	Mac:     "rq-service-macosx",
+	Unknown: "",
+}
+
+// DupeDetectionConfigs - dupe detection path of the supernode config
+var DupeDetectionConfigs = []string{
+	"dupe_detection_input_files",
+	"dupe_detection_support_files",
+	"dupe_detection_output_files",
+	"dupe_detection_processed_files",
+	"dupe_detection_rare_on_internet",
+	"mobilenet_v2_140_224",
+}
+
+// DupeDetectionSupportDownloadURL - The URL of dupe detection support files
+var DupeDetectionSupportDownloadURL = []string{
+	"https://download.pastel.network/machine-learning/dupe_detection_image_fingerprint_database.zip",
+	"https://download.pastel.network/machine-learning/keras_dupe_classifier.model.zip",
+	"https://download.pastel.network/machine-learning/xgboost_dupe_classifier.zip",
+	"https://download.pastel.network/machine-learning/nsfw_mobilenet_v2_140_224.zip",
+}
+
+// DupeDetectionMobileNetDownloadURL - The URL of dupe detection mobile net files
+var DupeDetectionMobileNetDownloadURL = "https://download.pastel.network/machine-learning/nsfw_mobilenet_v2_140_224.zip"
+
+// DupeDetectionSupportFilePath - The target path for downloading dupe detection support files
+var DupeDetectionSupportFilePath = "dupe_detection_support_files"
+
+// GetVersionSubURL returns the sub url concerned with version info
+func GetVersionSubURL(version string) string {
+	var versionSubURL string
+	switch version {
+	case "latest":
+		{
```

Comment:
you don't need code blocks inside cases. its redundant.

Reply:
@ayk was written the block inside cases. so I make the cases as a block.